editors/kate/Jakt.xml

```diff
@@ -0,0 +1,222 @@
+            <item>unsafe</item>
+            <item>try</item>
+            <item>catch</item>
+            <item>cpp</item>
+        </list>
+        
+        <!-- jaktWordOperator keywords  -->
+        <list name="jaktWordOperator">
+            <item>not</item>
+            <item>and</item>
+            <item>or</item>
+            <item>as</item>
+            <item>in</item>
+        </list>
+        
+        <!-- jaktVarDecl keywords  -->
+        <list name="jaktVarDecl">
+            <item>mut</item>
+            <item>let</item>
+            <item>anon</item>
+            <item>raw</item>
+        </list>
+        
+        <!-- jaktType keywords  -->
+        <list name="jaktType">
+            <item>String</item>
+            <item>i8</item>
+            <item>i16</item>
+            <item>i32</item>
+            <item>i64</item>
+            <item>u8</item>
+            <item>u16</item>
+            <item>u32</item>
+            <item>u64</item>
+            <item>f32</item>
+            <item>f64</item>
+            <item>bool</item>
+            <item>c_int</item>
+            <item>c_char</item>
+            <item>usize</item>
+            <item>void</item>
+        </list>
+        
+        <!-- jaktConstant keywords  -->
+        <list name="jaktConstant">
+            <item>this</item>
+        </list>
+        
+        <!-- jaktStructure keywords  -->
+        <list name="jaktStructure">
+            <item>struct</item>
+            <item>class</item>
+            <item>enum</item>
+            <item>namespace</item>
+            <item>trait</item>
+        </list>
+
+        <!-- jaktVisModifier keywords -->
+        <list name="jaktVisModifier">
+            <item>public</item>
+            <item>private</item>
+        </list>
+        
+        
+        <!-- Start Contexts -->
+        <contexts>
+            <!-- Normal text context -->
+            <context attribute="NormalText" lineEndContext="#pop" name="NormalText">
+                <DetectSpaces />
+                <!-- Start handling multiple-line comments : Comment -->
+                <StringDetect attribute="Comment" context="Comment" String="/*" beginRegion="comment"/>
+                
+                <!-- Start handling single-line comment -->
+                <RegExpr attribute="Comment" context="assign" String="\/\/.*" firstNonSpace="false" />
+                
+                <!-- Start handling multiple-line string : jaktDoubleQuot -->               
+                <DetectChar attribute="String" context="jaktDoubleQuot" char="&quot;" />
+                
+                <!-- Start handling multiple-line string : jaktSingleQuot -->               
+                <DetectChar attribute="String" context="jaktSingleQuot" char="'" />
+        
+                <!-- Start handling JaktFunction -->
+                <RegExpr attribute="JaktFunction" context="assign" String="\w+[a-z-A-Z1-9](?=\s*\()" firstNonSpace="false" />
```

Comment:
This will break with explicit generic parameters, though none of the other highlighters can accurately detect this either, so not a merge blocker.

Reply:
I work on it to prevent a break on explicit generic parameters ;)